lib/improver/tests/acceptance/test_combine.py

```diff
@@ -50,7 +50,7 @@ def test_basic(tmp_path):
             "--new-name=cloud_area_fraction",
             low_cloud_path,
             medium_cloud_path,
-            output_path]
+            f"--output={output_path}"]
```

Comment:
It's fairly minor, but as a way of trying to standardise things a little more, we'd also like to try to have all of our arguments space separated, i.e: converting `--x=y` to `--x y`. Would it be possible to make this change here as well, please?

Reply:
I've changed this, but the result is a bit more messy as a blank space doesn't work, you have to separate the two halves of the argument into separate strings in the argument list. `"--output", f"{output_path"}`. Are you sure this is preferable?